src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -72,34 +88,96 @@ export class CdkMenuItemTrigger implements OnDestroy {
   /** Open/close the attached menu if the trigger has been configured with one */
   toggle() {
     if (this.hasMenu()) {
-      this.isMenuOpen() ? this._closeMenu() : this._openMenu();
+      this.isMenuOpen() ? this.closeMenu() : this.openMenu();
+    }
+  }
+
+  /** Open the attached menu. */
+  openMenu() {
+    if (!this.isMenuOpen()) {
+      this.opened.next();
+
+      this._overlayRef = this._overlay.create(this._getOverlayConfig());
```

Comment:
This will recreate the overlay every time you open it. You can reuse it like this:

```
if (!this._overlayRef) {
  this._overlayRef = this._overlay.create(this._getOverlayConfig());
}

this._overlayRef.attach(this._getPortal());
```